cpp/test/CMakeLists.txt

```diff
@@ -22,7 +22,8 @@ set(COMMON_TEST_LINK_LIBRARIES
   CUDA::cusparse
   CUDA::cufft
   rmm::rmm
-  raft::raft
+  raft
+  raft_distance
```

Comment:
```suggestion
  raft::raft
  raft::raft_distance
```

I would've thought the namespace is needed here?

Reply:
Indeed, the CI timeout is telling me something's not working as expected. Looking into this